pkg/reporter/text.tmpl

```diff
@@ -0,0 +1,9 @@
+
+Passed: {{GreenInt .Summary.RuleResults.PASS}}, Failed: {{RedInt .Summary.RuleResults.FAIL}}, Waived: {{CyanInt .Summary.RuleResults.WAIVED}}
+{{range $ruleResults := .FailuresByRule}}{{if $ruleResults.Results}}
+{{Cyan $ruleResults.RuleID}}: {{Bold $ruleResults.RuleSummary}} {{Severity $ruleResults.RuleSeverity}}
```

Comment:
Similar comment to above - we might want to show the rule name if the ID is `""`.

Reply:
Done